apps/server/src/modules/tool/external-tool/uc/external-tool.uc.spec.ts

```diff
@@ -180,6 +182,67 @@ describe('ExternalToolUc', () => {
 
 			expect(result).toEqual(externalTool);
 		});
+
+		describe('when tool has no logo url', () => {
+			const setup2 = () => {
```

Comment:
setup2 sounds bad. rename to something more fitting, maybe "setupLogo" like further down?